packages/util/package.json

```diff
@@ -23,7 +23,7 @@
     "build": "npm run build:node && npm run build:browser",
     "build:node": "../../config/cli/ts-build.sh node",
     "build:browser": "../../config/cli/ts-build.sh browser",
-    "prepublishOnly": "npm run clean && npm run build && npm run test",
+    "prepublishOnly": "../../config/cli/prepublish.sh",
```

Comment:
Ah, really nice to see that we start to leverage now this tighter config integration. 🙂